composer.json

```diff
@@ -38,7 +38,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "6.3-dev"
+            "dev-master": "7.0-dev"
```

Comment:
Shouldn't be `7.1`?

Reply:
No, this does not refer to the PHP version.